ironfish/src/telemetry/telemetry.ts

```diff
@@ -184,6 +184,41 @@ export class Telemetry {
         type: 'integer',
         value: this.metrics.memPoolSize.value,
       },
+      {
```

Comment:
I think this holding branch wasn't originally intended for review, so my comments here might not really be relevant. But I think in the future I'd prefer spending a little extra effort to put the core logic changes in a standalone PR into staging (behind a config if necessary). Then these telemetry, getStatus, and metrics changes could be separated out. I'd have to think more about the best way to split these out, but it does get tougher to provide quality review on a 1000s line PR, plus some of these issues may have already been commented on in the originating PRs.

Reply:
Yea agreed telemetry and getStatus changes could have gone in after this